packages/amplify-graphql-searchable-transformer/src/graphql-searchable-transformer.ts

```diff
@@ -299,14 +302,7 @@ export class SearchableModelTransformer extends TransformerPluginBase {
 
     const parameterMap = createParametersInStack(context.stackManager.rootStack);
 
-    const nodeToNodeEncryption = this.apiName
-      ? shouldEnableNodeToNodeEncryption(this.apiName, context.filepaths.findProjectRoot(), context.filepaths.getCurrentCloudBackendDirPath())
-      : { enabled: false, log: undefined };
-    if (nodeToNodeEncryption.log) {
-      this.warn(nodeToNodeEncryption.log);
-    }
-
-    const domain = createSearchableDomain(stack, parameterMap, context.api.apiId, nodeToNodeEncryption.enabled);
+    const domain = createSearchableDomain(stack, parameterMap, context.api.apiId, this.options?.enableNodeToNodeEncryption ?? false);
```

Comment:
Is the default value set to 'false' to handle apps created before nodeToNodeEncryption change?

Reply:
It's set to false because we want to ensure that customers ONLY enable nodeToNode if it was already enabled previously because disabling it is lossy, we don't want to toggle it on for customers accidentally.

So the behavior (preserved from existing) is:

If override is set (e.g. by manual action), then respect override.
If existingBackendStack contains value, respect existing value.
Else: false.